### DISCUSSION TITLE
An issue with updating the list of vulnerabilities

### DISCUSSION THREAD

**Discussion by redduxi** [DependencyTrack/dependency-track#3045] [Q&A] [answered]
Hello everybody,

Recently, I encountered a problem with updating the list of vulnerabilities in the dependency-track tool.
I installed the tool locally and also on an EC2 instance. In both cases, I configured absolutely identical configurations for synchronization with vulnerability databases. I also enabled Snyk analyzer.
However, the list of vulnerabilities is different. A tool installed locally in the list has significantly more vulnerabilities than a tool installed on an instance.

Does someone know what the problem might be and how to solve it?

Thanks

**Comment by syalioune** [comment]
Hello  @redduxi 
Can you provide more inputs like ?
- Application logs
- Detailed configuration
- Actual vulnerabilities count on both setup
- Example of missing vulnerabilities

**Reply by redduxi** [reply]
Hello @syalioune, 

I'm unsure where to look at the logs, but I'm attaching all the other information.

Configuration:
<img width="1319" alt="3" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/587ef0a1-c7c6-4ac7-ae88-4e53316e629b">
<img width="1317" alt="4" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/06619acd-7f5d-4e66-b85b-39cdb0ee8d91">
<img width="1319" alt="5" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/41feadd9-547b-4f5a-9aac-5855d80b49bc">
<img width="1312" alt="6" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/b13692b1-873e-417a-a66c-1c2528412efa">
<img width="1315" alt="7" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/dc18b9c0-693f-4c6b-b82a-9f091dec93d3">
<img width="1321" alt="8" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/8ef8c435-2156-436b-93a1-6d2dcec9363f">
The configurations are identical in both cases.

Number of vulnerabilities on localhost:
<img width="1372" alt="10" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/47917568-1578-476c-ae30-7f35d39da9ec">

Number of vulnerabilities on the instance:
<img width="1373" alt="9" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/0a63a950-e485-46d4-bd33-212237d5e0a4">

Example of missing vulnerabilities:
<img width="1376" alt="11" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/cf1fac84-4754-4458-b7eb-30ab2ebf955b">
<img width="1372" alt="12" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/bf2560b4-e0af-492d-af49-a2b1209deadf">

I can provide a longer list of missing vulnerabilities if needed.

Thanks

**Reply by syalioune** [reply, marked as answer]
Thank you.

> I'm unsure where to look at the logs, but I'm attaching all the other information.

You can find the server logs in `${user.home}/.dependency-track/` folder. Those will really be helpful as the differences could likely due to an error in the asynchronous tasks that performs external databases synchronization.
